File: CH1-input-process-output/EX6-sales-tax/salesTax.js
const prompt = require("prompt-sync")();

var stateTax = 0.04,
  countryTax = 0.02,
  totalStateTax,
  purchase,
  totalSalesTax,
  totalSale;

// Input
purchase = Number(prompt("Amount Purchase : $"));

// Process
totalStateTax = purchase * stateTax;
totalCountryTax = purchase * countryTax;
totalSalesTax = totalStateTax + totalCountryTax;
totalSale = purchase + totalSalesTax;

// Output
console.log("Purchase          : $" + purchase);
console.log("State Sales Tax   : $" + totalStateTax);
console.log("Country Sales Tax : $" + totalCountryTax);
console.log("Total Sales Tax   : $" + totalSalesTax);
console.log("Total Sale        : $" + totalSale);

File: CH1-input-process-output/EX6-sales-tax/salesTax.py
stateTax = 0.04
countryTax = 0.02

# Input
purchase = int(input("Amount Purchase : $"))

# Process
totalStateTax = purchase * stateTax
totalStateTax = int(totalStateTax)
totalCountryTax = purchase * countryTax
totalCountryTax = int(totalCountryTax)
totalSalesTax = totalStateTax + totalCountryTax
totalSale = purchase + totalSalesTax

# Output
print("Purchase          : $" + str(purchase))
print("State Sales Tax   : $" + str(totalStateTax))
print("Country Sales Tax : $" + str(totalCountryTax))
print("Total Sales Tax   : $" + str(totalSalesTax))
print("Total Sale        : $" + str(totalSale))
